client/src/pages/admin/users/UserModal.tsx

```diff
@@ -101,6 +110,26 @@ export default function UserModal({ selectedUserID, onClose }: UserModalProps) {
     });
   };
 
+  const handleDeleteUserClicked = () => {
+      let fName = getUserResult.data.user.firstName;
+      let lName = getUserResult.data.user.lastName;
+      let privilege = getUserResult.data.user.privilege;
+      if (privilege == Privilege.ADMIN) {
```

Comment:
Let's just hide the delete user button from labbies, that way we don't have to do a privilege check here.